client/src/language/diagnosticsSupport.ts

```diff
@@ -143,6 +146,26 @@ const checkIsAlwaysIgnoredDiagnostic = async (
   return false
 }
 
+const checkIsIgnoredInlinePythonDiagnostic = async (
+  diagnostic: vscode.Diagnostic,
+  originalTextDocument: vscode.TextDocument,
+  adjustedRange: vscode.Range
+): Promise<boolean> => {
+  if (
+    !hasSourceWithCode(diagnostic, 'Flake8', 'E303') && // Too many blank lines
+    !hasSourceWithCode(diagnostic, 'Flake8', 'E501') && // Line too long
+    !hasSourceWithCode(diagnostic, 'Pylint', 'W0104:pointless-statement') &&
+    !hasSourceWithCode(diagnostic, 'Pylint', 'W0106:expression-not-assigned')
+  ) {
+    return false
+  }
+
+  return await requestsManager.getIsPositionOnInlinePython(
```

Comment:
I am considering to include the ranges of all embedded regions into `EmbeddedLanguageDoc` and to use it instead of `getEmbeddedLanguageTypeOnPosition`. After all, it seems to be used by all "embedded language" functionalities.